src/components/ApplicationContext/ApplicationContext.tsx

```diff
@@ -133,6 +151,18 @@ export const ApplicationProvider: React.FC = () => {
     }
   }
 
+  const saveData = () => {
+    const data: SavedApplication = {
+      contact: contactFormData,
+      demographic: demographicFormData,
+      shortAnswer: shortAnswerFormData,
+      priorExperience: priorExperienceFormData,
+      connected: connectedFormData,
```

Comment:
This seems to not save the file locally (which is good since this is probably a security risk and we don't want to store that much data in the browser) but do we know have code that explicitly handles this or is it just working?

Reply:
This was definitely an oversight on my part. I did not consider the case with the resume, but it looks like the `File` object serializes to `{}` and doesn't fail when we do `JSON.stringify()` (at least on Chrome), but we should definitely explicitly remove that field.

I'm also noticing that this stores the error messages too, which is OK I guess unless we want to strip those out as well.

Reply:
We can probably just strip the error messages and file. If you want, we can modify the current interface to extend of the fields that you need for storing and retrieving.